account_cutoff_accrual_picking/README.rst

```diff
@@ -0,0 +1,17 @@
+# Account Cutoff Accrual Picking
+
+## Notes about migration to v.10
+Historically, this module manages accrued expenses and revenues based on stock-picking.
+But following changes up to v.10 in the odoo database schema it can not function as before.
+
+Now instead of basing itself on the stock_picking to generate the calculation, it works directly with sale_order_line and purchase_order_line.
+
+Although a cut-off date can be precised for generating the accruals, it can not work correctly with the module as it is.
+
+## Configuration
+
+
+## Usage
+
+
+
```

Comment:
Need to use this template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst